ensenso_camera/scripts/request_data

```diff
@@ -23,10 +23,14 @@ def main():
 
     goal.publish_results = True
 
-    request_data_client = actionlib.SimpleActionClient("request_data", RequestDataAction)
+    request_data_client_name = rospy.get_param("~namespace", "") + "/request_data"
```

Comment:
```suggestion
    request_data_client_name = rospy.get_param("~namespace", "/") + "request_data"
```

Die Idee von den relativen Pfaden ist eigentlich, dass man die im Launchfile in den selben Namespace packen kann oder mit der Umgebungsvariable. Siehe hier: http://wiki.ros.org/Remapping%20Arguments#A.22Pushing_Down.22

Wir können den Parameter auch unterstützen, aber standardmäßig sollte der Pfad überall relativ sein.

Reply:
Ah Moment, mir ist grade aufgefallen, dass meine vorgeschlagene Änderung gar nicht reicht. Das macht den Pfad ja trotzdem absolut, wenn man den Parameter nicht angibt.